Exercise_C/index.css

```diff
@@ -2,3 +2,34 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+
+.container{
+    width: 500px;
+    height: 200px;
+    background-color: black;
+    flex-direction: row;
+    display: flex;
+    justify-content: space-between;
+}
+
+.box{
+    width: 100px;
+    height: 100px;
+}
+
+
+.blue{   
+    background: blue;
+
+}
+
+.red{  
```

Comment:
Consider the order of your boxes, and perhaps use flex-wrap property instead of using margin to push the element down.

Though you did recreate the image well so good work!